static/app/views/monitors/components/overviewTimeline/jobTickTooltip.tsx

```diff
@@ -0,0 +1,106 @@
+import {ComponentProps, Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import DateTime from 'sentry/components/dateTime';
+import Text from 'sentry/components/text';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {
+  JobTickData,
+  TimeWindow,
+} from 'sentry/views/monitors/components/overviewTimeline/types';
+import {CheckInStatus} from 'sentry/views/monitors/types';
+import {getColorsFromStatus, statusToText} from 'sentry/views/monitors/utils';
+
+import {timeWindowData} from './utils';
+
+interface Props extends Omit<ComponentProps<typeof Tooltip>, 'title'> {
```

Comment:
```suggestion
interface Props extends Omit<TooltipProps, 'title'> {
```

you can get `TooltipProps` from `Tooltip` now

Reply:
oooh nice